luigi/interface.py

```diff
@@ -238,8 +238,6 @@ def parse_task(self, cmdline_args=None):
 
         _add_global_parameters(parser)
 
-        task_names = Register.task_names()
```

Comment:
Original message was:

```
luigi/interface.py:241:9: F841 local variable 'task_names' is assigned to but never used
```

`flake8` do only static code analysis. If `Register.task_names()` do something except returning task names, removing that line could be potentially dangerous.